frontend/src/main/components/Commons/CommonsOverview.js

```diff
@@ -17,6 +17,7 @@ export default function CommonsOverview({ commons, currentUser }) {
                     <Col>
                         <Card.Title>Today is day {commons.day}! </Card.Title>
                         <Card.Text>Total Players: {commons.totalPlayers}</Card.Text>
+                        <Card.Text>Current milk price: ${commons.milkPrice}</Card.Text>
```

Comment:
I'm surprised that there is no test for this.  That should probably be added in a future PR.